src/maintainer/adding_pkgs.rst

```diff
@@ -761,6 +758,22 @@ For some languages, the community provides tools which can automate this process
 
    The correct and automated packaging of dependency licenses is an ongoing discussion. Please feel free to add your thoughs to `this <https://github.com/conda-forge/conda-forge.github.io/issues/1052>`__ discussion. 
 
+Extra
+-----
+
+Recipe Maintainer
+..................
+
+A maintainer is an individual who is responsible for maintaining and updating one or more feedstock repositories and packages as well as their future versions. They have push access to the feedstock repositories of only the packages they maintain and can merge pull requests into it.
```

Comment:
Comma needed after the first "packages".

Replace the "it" at the end of the paragraph with "them".